iopipe/contrib/trace/marker.py

```diff
@@ -8,5 +8,8 @@ def start(self, name):
     def end(self, name):
         self.timeline.mark('end:%s' % name)
 
-    def measure(self, name, start, end):
-        self.timeline.measure('measure:%s' % name, 'start:%s' % start, 'end:%s' % end)
+    def measure(self, name, start=None, end=None):
+        self.timeline.measure(
+            'measure:%s' % name,
+            'start:%s' % (start or name),
```

Comment:
@pselle In the JS implementation the `start` and `end` are marker labels, not numbers. They're interpolated into `start:${start}` and `end:${end}`. So if a user doesn't pass `start` and `end` to the JS implementation it searches for `start:undefined` and `end:undefined` which is probably not what we want as that could get a match if the plugin is misused, such as calling `start()` or `end()` without arguments. I'm trying to avoid that here (`start:Name` and `end:None`).

Reply:
Here's the equivalent implementation: https://github.com/iopipe/iopipe-plugin-trace/blob/master/src/index.js#L74

Reply:
Got it, thank you for the clarification